packages/app/src/components/inspector/AlignmentInspector.tsx

```diff
@@ -27,14 +27,22 @@ function AlignmentInspector(props: AlignmentInspectorProps) {
   const [, dispatch] = useApplicationState();
   return (
     <AlignmentInspectorContainer>
-      <SpaceEvenlyHorizontallyIcon />
-      <SpaceEvenlyVerticallyIcon />
-      <AlignLeftIcon onClick={() => dispatch('alignLeft')} />
-      <AlignCenterHorizontallyIcon />
-      <AlignRightIcon onClick={() => dispatch('alignRight')} />
-      <AlignTopIcon onClick={() => dispatch('alignTop')} />
-      <AlignCenterVerticallyIcon />
-      <AlignBottomIcon onClick={() => dispatch('alignBottom')} />
+      <SpaceEvenlyHorizontallyIcon
```

Comment:
Let's move this file to `/containers`, since it uses `dispatch`. I think originally I had planned to expose a bunch of callbacks as props, and that's why this component is in `/components`. But I like this approach better, so let's just move the file